app/main.py

```diff
@@ -1 +1,30 @@
-# write your code here
+class Animal:
+    alive = []
```

Comment:
The `alive` list is defined as a class attribute, which means it is shared across all instances of `Animal`. This could lead to unexpected behavior if multiple instances are created. Consider making `alive` an instance attribute instead.